### PR TITLE
Issue #SB-13565 chore: Creating new Keycloak clients: google-auth-android and trampoline-android

### PR DESCRIPTION
**Pull request by ahghatol**

### CODE REVIEW

File: ansible/roles/sunbird-auth-deploy/files/python-keycloak-0.12.0/clients.json [hunk 1]
```diff
@@ -0,0 +1,266 @@
+{
+    "clients" : [
+        {
+            "clientId": "google-auth-android",
+            "rootUrl": "https://dev.sunbirded.org",
```
Comment (SMYALTAMASH) [inline ansible/roles/sunbird-auth-deploy/files/python-keycloak-0.12.0/clients.json]: can the DNS be variablized and not hardcoded.

File: ansible/roles/sunbird-auth-deploy/tasks/deploy.yml [hunk 2]
```diff
@@ -156,3 +156,44 @@
...
+    name: "{{item}}"
+    update_cache: yes
+  with_items:
+    - python-pip
+    - python-setuptools
+
+- name: Copying python libraries
+  copy: src={{ role_path }}/files/python-keycloak-0.12.0 dest=/tmp/
+
+- name: Initialize python library to run keycloak paython script
```
Comment (SMYALTAMASH) [inline ansible/roles/sunbird-auth-deploy/tasks/deploy.yml]: Spelling mistakes like paython needs to be changed.

File: ansible/roles/sunbird-auth-deploy/tasks/deploy.yml [hunk 3]
```diff
@@ -156,3 +156,44 @@
...
+  shell: cd /tmp/python-keycloak-0.12.0 && python setup.py install
+
+- name: Save keycalok vars to json
```
Comment (SMYALTAMASH) [inline ansible/roles/sunbird-auth-deploy/tasks/deploy.yml]: spelling mistake keycalok need to be changed.

File: ansible/roles/sunbird-auth-deploy/tasks/deploy.yml [hunk 4]
```diff
@@ -156,3 +156,44 @@
...
+- name: Copy realm json file to tmp location
+  template: src="keycloak-realm.j2" dest="/tmp/keycloak-realm.json" mode="0644"
+
+- name: Copy clients file to tmp location
+  copy: src="files/python-keycloak-0.12.0/clients.json" dest="/tmp/clients.json" mode="0644"
+
+- name: Copy user manager roles file to tmp location
+  copy: src="files/python-keycloak-0.12.0/roles.json" dest="/tmp/roles.json" mode="0644"
+
+- name: Pausing for 100 seconds sometime for keycloak to run completely
```
Comment (SMYALTAMASH) [inline ansible/roles/sunbird-auth-deploy/tasks/deploy.yml]: Do we need to wait for 100 seconds?

File: ansible/roles/sunbird-auth-deploy/files/python-keycloak-0.12.0/clients.json (hunk 1 above)
Comment (ahghatol) [reply ansible/roles/sunbird-auth-deploy/files/python-keycloak-0.12.0/clients.json]: Fixed

File: ansible/roles/sunbird-auth-deploy/tasks/deploy.yml (hunk 4 above)
Comment (ahghatol) [reply ansible/roles/sunbird-auth-deploy/tasks/deploy.yml]: This is not needed now

**Pull request merged**